tracer/test/snapshots/AspNetCoreMinimalApisTests.WithFF.__path=__statusCode=200.verified.txt

```diff
@@ -31,14 +31,17 @@
       component: aspnet_core,
       datadog-header-tag: asp-net-core,
       env: integration_tests,
+      http.client_ip: 127.0.0.1,
       http.method: GET,
       http.request.headers.host: localhost:00000,
       http.request.headers.sample_correlation_identifier: 0000-0000-0000,
       http.response.headers.sample_correlation_identifier: 0000-0000-0000,
       http.response.headers.server: Kestrel,
+      http.route: {controller=home}/{action=index}/{id?},
```

Comment:
These minimal APIs _do_ have the `http.route` (as FF enabled) but no user agent, maybe we're not calling `DecorateWebSpan`?